main.tf

```diff
@@ -188,7 +189,7 @@ module "jump" {
   cloud_init        = data.template_cloudinit_config.jump.rendered
   create_public_ip  = var.create_jump_public_ip
 
-  depends_on = [module.nfs]
+  depends_on = [module.vnet]
```

Comment:
Can you provide info on changing this one from nfs -> vnet? The nfs server must be active before the jump box can configure.

Reply:
`vnet` to add explicit module dependency and issues encountered when testing. If jump vm depends on 'nfs` then I can add it back with a comment.

Reply:
In the future you can add both since `depends_on` does take a list. Also the fact the Jump server does require the NFS server to be active was the original reason for the nfs dependency.

For example:
`depends_on = [module.nfs, module.vnet]`